### ISSUE TITLE
Compile warning: compile on ventura warns that 'FSEventStreamScheduleWithRunLoop' is deprecated

### ISSUE THREAD

**Issue by AdamSharon** [fsnotify/fsevents#59]
### Which version of macOS are you using?

ProductName:            macOS
ProductVersion:         13.2.1
BuildVersion:           22D68

### Please describe the issue that occurred.
Compilation throws a warning:
```bash
# github.com/fsnotify/fsevents
cgo-gcc-prolog:452:2: warning: 'FSEventStreamScheduleWithRunLoop' is deprecated: first deprecated in macOS 13.0 - Use FSEventStreamSetDispatchQueue instead. [-Wdeprecated-declarations]
/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/System/Library/Frameworks/CoreServices.framework/Frameworks/FSEvents.framework/Headers/FSEvents.h:1138:1: note: 'FSEventStreamScheduleWithRunLoop' has been explicitly marked deprecated here
```

There is no functionality problem right now, as far as I see. I hope it does not mean a future update to MacOS will break it.

### Are you able to reproduce the issue? Please provide steps to reproduce and a code sample if possible.
Yes, on a machine that runs Ventura (13.0 and up) any compilation of a project that contains `fsevents v0.1.1` will throw the warning.

**Comment by arp242** [comment]
This library is essentially unmaintained (I'm not maintaining it, anyway). If you want a fix you'll have to send a patch.

**Comment by arp242** [comment]
Fixed via  #62.
